schemas/x509.yml

```diff
@@ -0,0 +1,154 @@
+---
+- name: x509
+  title: x509 Certificate
+  group: 2
+  description: These fields contain x509 certificate metadata.
+  type: group
+  reusable:
+    top_level: false
+    expected:
+      - file
+      - tls
+  fields:
+    - name: version_number
+      level: extended
+      type: long
+      description: Version of x509 format.
+      example: 3
+
+    - name: serial_number
+      level: extended
+      type: keyword
+      description: Unique serial number issued by the certificate authority.
+      example: 55:fb:b9:c7:de:bf:09:80:9d:12:cc:aa
+
+    - name: issuer.distinguished_name
+      level: extended
+      type: keyword
+      description: Distinguished name (DN) of issuing certificate authority.
+      example: C=US, O=DigiCert Inc, OU=www.digicert.com, CN=DigiCert SHA2 High Assurance Server CA
+
+    - name: issuer.common_name
+      level: extended
+      type: keyword
+      description: Common name (CN) of issuing certificate authority.
+      example: DigiCert SHA2 High Assurance Server CA
+
+    - name: issuer.organizational_unit
+      level: extended
+      type: keyword
+      description: List of organizational units (OU) of issuing certificate authority.
+      example: www.digicert.com
+
+    - name: issuer.organization
+      level: extended
+      type: keyword
+      description: Organizational units (O) of issuing certificate authority.
+      example: DigiCert Inc
+
+    - name: issuer.locality
+      level: extended
+      type: keyword
+      description: Locality name (L)
+      example: Mountain View
+
+    - name: issuer.state_or_province
+      level: extended
+      type: keyword
+      description: State or province name (ST, S, or P)
+      example: California
+
+    - name: issuer.country
+      level: extended
+      type: keyword
+      description: Country (C) code
+      example: US
+
+    - name: signature_algorithm
+      level: extended
+      type: keyword
+      description: Identifier for certificate signature algorithm. Recommend using names found in Go Lang Crypto library (https://golang.org/src/crypto/x509/x509.go).
```

Comment:
also, it might be helpful to find a permalink to the source code, since that's a file tied to the latest go version, it could always change in the future. I'm assuming you're talking about these identifiers?

https://github.com/golang/go/blob/fff7509d472778cae5e652dbe2479929c666c24f/src/crypto/x509/x509.go#L335-L351

Reply:
Good point about adjusting the URL so that it stays good over time.

A few thoughts, however.

If there's a better place to link to (not code, but a website for humans), that would be great. If not, that's fine and we can link to this code.

Ideally we avoid Asciidoc links, because these descriptions are not only used in the main docs, but also in code comments & other places that aren't asciidoc. So to keep the link short and legible, I'd recommend using a Git tag instead of a full hash, and potentially even skip the line numbers. So either:

https://github.com/golang/go/blob/go1.14/src/crypto/x509/x509.go

or 

https://github.com/golang/go/blob/go1.14/src/crypto/x509/x509.go#L337-L353

Reply:
I'll use the latter. An alternative, is we copy and paste those values here and mark them as "acceptable values".